theme_kaizen/templates/snippets/posts.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="snippet_posts" name="Posts">
+        <section class="snippet_kaizen_posts home-posts">
+            <div class="container">
+                <!-- Heading -->
+                <h3>Latest updates</h3>
+                <!-- Slider -->
+                <div class="home-posts__slider" contenteditable="false">
+                    <!-- Swiper -->
+                    <div class="swiper-container">
+                        <div class="swiper-wrapper">
```

Comment:
This one needs at least one swiper item, so that it can be duplicated by the user.